### PR TITLE
style(ui): Profile & Org page update

### PR DESCRIPTION
**Pull request by juliendonck**
Update the style of profiles and org to represent latest state in design.

Touches:
- Avatar component
- Icon.Badge -> Icon.Verified
- Org pages
- Profile pages
- List component (for centring lists)
- SidebarLayout (for centring lists)

Screenshot 2020-06-15 at 15 12 57
Screenshot 2020-06-15 at 15 12 33

### CODE REVIEW

File: ui/DesignSystem/Component/BigHeader.svelte [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  import { Avatar, Icon, Title, Text, Numeric } from "../Primitive";
+</script>
+
+<style>
+  .header {
+    display: flex;
+    flex-direction: column;
+    width: calc(100vw - var(--sidebar-width));
+    height: 261px;
```
Comment (cloudhead) [inline ui/DesignSystem/Component/BigHeader.svelte:14]: Where's this number from? Kind of a strange number :thinking:

File: ui/Screen/Org.svelte [hunk 2]
```diff
@@ -82,30 +82,21 @@
 
 <SidebarLayout
   dataCy="org-screen"
-  style="margin-top: calc(var(--topbar-height) + 33px)">
+  style="margin-top: calc(var(--topbar-height) + 200px)">
```
Comment (cloudhead) [inline ui/Screen/Org.svelte:85]: Can we put this `200px` in a var? It's also used in `.banner`, so changing one of those will break the other..

File: ui/DesignSystem/Component/BigHeader.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/BigHeader.svelte]: resolved in [80abf4a](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/80abf4a62ab6607f079bd6c6c8d840b554742fae)

File: ui/Screen/Org.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/Screen/Org.svelte]: resolved in [80abf4a](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/80abf4a62ab6607f079bd6c6c8d840b554742fae)

File: ui/DesignSystem/Component/BigHeader.svelte [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<script>
+  export let style = null;
+  export let data = null;
```
Comment (rudolfs) [inline ui/DesignSystem/Component/BigHeader.svelte:3]: This should probably be `entity` as it's either an `org` or `user`.

File: ui/DesignSystem/Component/BigHeader.svelte [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  export let variant = null; // profile | org
+
+  import { Avatar, Icon, Title, Text, Numeric } from "../Primitive";
```
[the remark this answers is not in the document]
Comment (juliendonck) [reply ui/DesignSystem/Component/BigHeader.svelte]: Addressed in [81a3378](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/81a33787568db5f5a6247813b280e58a7a590afa)

File: ui/DesignSystem/Component/BigHeader.svelte [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    flex-direction: column;
+    align-self: center;
+  }
+
+  .user {
+    display: flex;
+    align-items: center;
+  }
+
+  .id {
```
[the remark this answers is not in the document]
Comment (juliendonck) [reply ui/DesignSystem/Component/BigHeader.svelte]: Addressed in [81a3378](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/81a33787568db5f5a6247813b280e58a7a590afa)

File: ui/DesignSystem/Component/BigHeader.svelte [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        <div class="id">
+          <Text variant="tiny" style="margin-right: 4px;">Radicle ID</Text>
+          <Numeric variant="small">{data.shareableEntityIdentifier}</Numeric>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="action-bar">
+    <slot name="left" />
+    <div class="right">
```
[the remark this answers is not in the document]
Comment (juliendonck) [reply ui/DesignSystem/Component/BigHeader.svelte]: Addressed in [81a3378](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/81a33787568db5f5a6247813b280e58a7a590afa)

File: ui/DesignSystem/Component/BigHeader.svelte (hunk 3 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/BigHeader.svelte]: Addressed in [81a3378](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/81a33787568db5f5a6247813b280e58a7a590afa)

**Pull request merged**